src/client.js

```diff
@@ -702,7 +702,63 @@ const fileUploader = (client) => {
                 else
                   throw CampaignException.AEM_ASSET_UPLOAD_FAILED(ex, ex.statusCode);
             }
+        },
+
+        /**
+         * Exposed public method of the fileUploader api,
+         * specifically designed to download files from the 'upload' folder of the Campaign instance
+         * @param {string} md5 md5 of the file content
+         * @param {string} ext (original) file extension
+         * @param {Object | undefined} options
+         * @param {string} options.fileName to rename the name in save as dialog of web browser 
+         * (force this dialog) must be in UTF-8
+         * @param {string} options.contentType change the content-type of the response (to help browser to
+         * handle this file) expl : image/png, text/plain;charset=ISO-8859-1
+         * @returns {Promise<File(string)>}
+         */
+        download: async (md5, ext, options) => {
+        const response = { data: null, errors: null };
```

Comment:
This does not seem to be used. Can you remove the line?

Reply:
Done